sdk/nodejs/runtime/closure/createClosure.ts

```diff
@@ -172,11 +165,28 @@ interface ContextFrame {
     capturedModule?: { name: string, value: any };
 }
 
+interface ClosurePropertyDescriptor {
+    /** name of the property for a normal property. either 'name' or 'symbol' will be present.  but not both. */
+    name?: string;
+    /** symbol-name of the property.  either 'name' or 'symbol' will be present.  but not both. */
+    symbol?: symbol;
```

Comment:
In what cases would `symbol` be set while `name` is not?

Reply:
for a symbolicly named property.  i.e.  `{ [mySym]: 0 }`.